external/qrcodegen/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+project(libqrcodegen)
+
+add_library(qrcodegen STATIC QrCode.cpp)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```

Comment:
Does -fPIC not get passed already ? If it doesn't, do the security flags get not passed either ?

Reply:
I had to add `-fPIC` to get Travis CI cross compilation working.

The security flags probably don’t get passed either.

Reply:
I don't either, I just hack at it till it works :D It can stay that way, I or someone will fix it later.